packages/FieldTypes/src/components/Numeric/Numeric.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Container from "../Container";
+import types from "../../types";
+
+// This let mimic the use of Field.types.align.LEFT in documentation and d.ts files
+// not other reason to have it
+const FieldTypes = {};
+FieldTypes.types = types;
+
+function formatNumber({ number, locale, options = {} }) {
+  return Intl.NumberFormat(locale, options).format(number);
+}
+
+export function withDecimalSeparatorOnly({ number, locale, options = {} }) {
+  return new Intl.NumberFormat(locale, {
+    useGrouping: true,
+    ...options,
+  })
+    .formatToParts(number)
+    .filter(chunk => chunk.type !== "group")
+    .map(chunk => chunk.value)
+    .join("");
+}
+
+export default function Numeric(props) {
+  const { align, number: propNumber, currency: currencySymbol, intl, color, displayOnlyDecimals, ...moreProps } = props;
+  const number = Number(propNumber);
+  const i18n = useI18n();
+
+  if (Number.isNaN(number)) {
+    console.warn(`string|number ${propNumber} is Not a Number`);
+  }
+
+  const currency = currencySymbol ? { style: "currency", currency: currencySymbol } : {};
+  const decimalNumbers =
+    number % 1 === 0 ? { minimumFractionDigits: 0, maximumFractionDigits: 0 } : { minimumFractionDigits: 1 };
+
+  const i18nNumber =
+    "Intl" in window
+      ? formatNumber({ number, locale: i18n.locale, options: { ...currency, ...decimalNumbers, ...intl } })
+      : propNumber;
+
+  if (displayOnlyDecimals) {
+    const numberWithDecimalsOnly = withDecimalSeparatorOnly({
+      number,
+      locale: i18n.locale,
+      options: { ...currency, ...decimalNumbers, ...intl },
+    });
+    return (
+      <Container align={align} color={color} {...moreProps} data-pka-anchor="data-fields-numeric">
+        {typeof cell === "function" ? propNumber(numberWithDecimalsOnly) : numberWithDecimalsOnly}
+      </Container>
+    );
+  }
+
+  return (
+    <Container align={align} color={color} {...moreProps} data-pka-anchor="data-fields-numeric">
+      {typeof cell === "function" ? propNumber(i18nNumber) : i18nNumber}
+    </Container>
+  );
+}
+
+Numeric.propTypes = {
+  /**
+   * The cell value to be localize
+   */
+  number: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
```

Comment:
The PR description had this as `cell` not `number` and I came to complain, but `number` is better.  But... since there will be other "Field Type" components too, maybe we could use something like `value` which would allow a consistent API across other types.  This might be useful for dynamic type cells where we could use the `as` prop in a `map()` to set what `Field Type` component is used.

Reply:
sound good to me, yes I updated the readme and the code after reading my original PR, forgot to update the PR description.